### PR TITLE
[WIP]support --cluster-number for multi-cluster test

### PR DESCRIPTION
**Pull request by irisdingbj**

### CODE REVIEW

**Review by brian-avery — APPROVED** [review]

File: prow/lib.sh [hunk 1]
```diff
@@ -253,17 +257,22 @@ EOF
         done
       done
     }
+
     # Network 1 contains clusters 1, 2 and 3
```
Comment (howardjohn) [inline prow/lib.sh:261]: I don't understand how we can have a variable number of clusters but then this is fixed?

**Review by howardjohn — CHANGES_REQUESTED** [review]

**Review by nmittler — COMMENTED** [review]
I'm not sure this PR is the right path forward.

@su225 is working on a more flexible way of specifying a variable number of clusters (see https://github.com/istio/common-files/pull/310). I think we should wait for that work to complete and then just use that for the config cluster job.

FYI @stevenctl @howardjohn

**Comment by irisdingbj** [comment]
@nmittler Thanks for this pointer.  basically we need a way to specify different cluster setting for at least different prow jobs.   I left a question in https://github.com/istio/istio/pull/27248

**Comment by nmittler** [comment]
@irisdingbj 

> @nmittler Thanks for this pointer. basically we need a way to specify different cluster setting for at least different prow jobs. I left a question in #27248

Yeah, that's basically what that PR is intended to support.

FYI @su225

**Comment by irisdingbj** [comment]
Thanks  @su225 @nmittler for offering function needed in #27248. 
Thanks for the review @brian-avery  @howardjohn  since #27248 offer the related function. Let me close this PR.

**Pull request closed without merging**